packages/react-jsx-highcharts/src/components/AxisProvider/index.js

```diff
@@ -29,15 +30,16 @@ export default function provideAxis(WrappedComponent) {
       const id = this.props.axisId || this.props.id;
       if (!id) return null;
 
+      const chart = this.props.getChart();
       const axis = this.props.get(id);
```

Comment:
added a `boundContextHelper` to `getBoundChartMethod`. The code got much cleaner indeed, thanks =)